### ISSUE TITLE
Design Intent - Accessibility Feedback - IIR, School Administrators, Resources for Schools

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#73714]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- No big concerns about what you presented, but lots of opportunities to clean up what's there now!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** Make sure the heading hierarchy makes sense (the current one doesn't). The test here goes like this: Pretend you're back in 9th grade English class and make an outline of the page. Each heading should be an item in the outline, and they should be nested according to their relationship to each other --- eg. a subsection heading nested inside a broader related section. The nesting corresponds to your heading levels, with H1 the page title, H2 each big section, H3 each subsection, etc. Be careful about accordions too, as the accordion title (trigger to open/close it) counts in your heading hierarchy. Also be careful about the right-hand sidebar, which also lives in the same heading hierarchy (following the main content).
- **Should:** Wherever possible, use existing design system components. There's some weird stuff in the current version, like a right chevron on a link that's an `img` with alt text "right arrow." That's not meaningful information, and the appropriate design system components help avoid issues like that.

**Comment by DavidPearlUX** [comment]
Hello @briandeconinck ,

Thank you for your feedback. I have aligned with my team and we propose the following changes to address your concern:

1. The updated page now has [Dev annotations](https://www.figma.com/file/jxsiqfLfxf4uulELWxpGTF/Resources-for-Schools?type=design&node-id=534%3A1092&mode=dev) which provides clear direction on intended header structure. 
2. As far as the chevron images, our team noticed them too. Dev has reassured me that as we work through our overhaul we will touch each section, and that will be our opportunity to replace the custom HTML with link components.

Also tagging in @emcruer who will be supporting this ticket while I am abroad over the next few weeks.

Thanks, 
David
